src/Data/Integer/Properties.agda

```diff
@@ -2064,3 +2597,17 @@ Please use _<_ instead."
 "Warning: [1+m]*n≡n+m*n was deprecated in v1.2.
 Please use suc-* instead."
 #-}
+
+-- Version 1.5
+
+neg-mono-<-> = neg-mono-<
```

Comment:
Any deprecated names need to be added to the CHANGELOG as well.

https://github.com/agda/agda-stdlib/blob/master/CHANGELOG.md#deprecated-names